config/default.yml

```diff
@@ -119,6 +119,15 @@ Rails/BulkChangeTable:
   Include:
     - db/migrate/*.rb
 
+Rails/ContentTag:
+  Description: 'Use `tag` instead of `content_tag`.'
+  Reference:
+    - 'https://github.com/rails/rails/issues/25195'
+    - 'https://api.rubyonrails.org/classes/ActionView/Helpers/TagHelper.html#method-i-content_tag'
+  Enabled: false
```

Comment:
I think this cop can be enabled by default. Are there any points to worry about?

Reply:
I worried about that `<br></br>` becomes `<br>`, but now I think it's trivial.
I will make it enabled.